guides/common/assembly_managing-isos-and-files.adoc

```diff
@@ -1,8 +1,4 @@
-[id="Managing_ISO_Images_{context}"]
-= Managing ISO Images
-
-You can use {Project} to store ISO images, either from Red{nbsp}Hat's Content Delivery Network or other sources.
-You can also upload other files, such as virtual machine images, and publish them in repositories.
+include::modules/con_managing-iso-images.adoc[]
```

Comment:
Mark -- see my comment for `assembly_synchronizing-content-between-foreman.adoc`